src/plugins/debug/configuration.rs

```diff
@@ -58,9 +62,10 @@ impl PhysicsDebugConfig {
         }
     }
 
-    /// Disables all debug rendering for this entity.
+    /// Disables all debug rendering.
     pub fn none() -> Self {
         Self {
+            enabled: false,
```

Comment:
This feels like a potential footgun, since users might expect e.g. `PhysicsDebugConfig::none().with_collider_color(Color::WHITE)` to work.

I'm not sure what the best approach would be, but maybe we could just use `enabled: true` here, but clarify the doc comment to be:

```rust
/// Creates a [`PhysicsDebugConfig`] configuration with debug rendering enabled but all options turned off.
```

or something like that.

Reply:
Looks good now, thanks